controllers/api/user-routes.js

```diff
@@ -44,6 +44,19 @@ router.post("/login", async (req, res) => {
 });
 
 // Sign-Up Endpoint(s)
+router.post('/', async (req, res) => {
+    try {
+        const userData = User.create(req.body);

```

Comment:
You'll likely need to add a check that the user creation was successful. Something like the pseudocode below:

`if(userData) {`
`  // run the session.save stuff`
`}`
`else {`
`  // return a 400 error`
`}`